File: heartbeat/src/main/java/com/expresspay/heartbeat/models/Payload.java
package com.expresspay.heartbeat.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Payload implements Serializable {

    @SerializedName("device-id")
    public String deviceId = "";

    @SerializedName("version")
    public String apiVersion = "";

    @SerializedName("appid")
    public String appId = "";

    @SerializedName("appname")
    public String appName = "";

    @SerializedName("appver")
    public String appVer = "";

    @Override
    public String toString() {
        return "Payload{" +
                "version='" + apiVersion + '\'' +
                ", deviceId='" + deviceId + '\'' +
                ", appId='" + appId + '\'' +
                ", appName='" + appName + '\'' +
                ", appVer='" + appVer + '\'' +
                '}';
    }
}

File: heartbeat/src/main/java/com/samdev/heartbeat/models/Payload.kt
package com.samdev.heartbeat.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Payload(
    @SerializedName("device-id")
    var deviceId: String = "",

    @SerializedName("version")
    var apiVersion: String = "",

    @SerializedName("appid")
    var appId: String = "",

    @SerializedName("appname")
    var appName: String = "",

    @SerializedName("appver")
    var appVer: String = ""

) : Serializable {
    override fun toString(): String {
        return "Payload{" +
                "version='" + apiVersion + '\'' +
                ", deviceId='" + deviceId + '\'' +
                ", appId='" + appId + '\'' +
                ", appName='" + appName + '\'' +
                ", appVer='" + appVer + '\'' +
                '}'
    }
}